packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -53,6 +53,12 @@ export class Autosuggest {
   @Prop()
   suggestOnFocus = false;
 
+  /**
+   * A function provided by the consumer of the autosuggest component, that returns an array of `AutosuggestMarkItem`s
+   */
+  @Prop()
+  mark?: (text: string, type: "value" | "type" | "extra", extraIndex?: number) => AutosuggestMarkItem[];
```

Comment:
Zoals overlegd, signature van de markeerfunctie uitbreiden met `suggestion: Suggestion`.
```suggestion
  mark?: (suggestion: Suggestion, text: string, type: "value" | "type" | "extra", extraIndex?: number) => AutosuggestMarkItem[];
```

Reply:
Ook gedaan